package.json

```
@@ -1,101 +1,10 @@
-    "build": "pnpm run build-bundles && pnpm run build-types && pnpm run build-storybook",
-    "build-bundles": "vite build --mode production",
-    "build-types": "tsc --project tsconfig.build.json",
-    "serve": "vite preview --host",
-    "check": "concurrently -g pnpm:check-*",
-    "check-svelte": "svelte-check --tsconfig ./tsconfig.json",
-    "check-vue": "vue-tsc --project ./playground/tsconfig.json",
-    "check-lint": "eslint --quiet \"./**/*.@(js|cjs|ts|svelte)\"",
-    "check-format": "pnpm run _prettier --check",
-    "format": "pnpm run _prettier --write",
-    "_prettier": "prettier \"./**/*.@(js|cjs|ts|svelte|md|mdx|yml|json)\"",
-    "test": "playwright test",
-    "test-dev": "playwright test --debug",
-    "storybook": "storybook dev -p 6006",
-    "storybook-docs": "storybook dev --docs --no-manager-cache",
-    "build-storybook": "storybook build --docs -o prime"
-  },
-  "devDependencies": {
-    "@floating-ui/dom": "^1.2.6",
-    "@floating-ui/vue": "^1.0.0",
-    "@mdi/js": "^7.2.96",
-    "@playwright/test": "^1.32.3",
-    "@storybook/addon-a11y": "^7.0.7",
-    "@storybook/addon-actions": "^7.0.7",
-    "@storybook/addon-docs": "^7.0.7",
-    "@storybook/addon-essentials": "^7.0.7",
-    "@storybook/addon-interactions": "^7.0.7",
-    "@storybook/addon-links": "^7.0.7",
-    "@storybook/addons": "^7.0.7",
-    "@storybook/blocks": "^7.0.7",
-    "@storybook/html": "^7.0.7",
-    "@storybook/html-vite": "^7.0.7",
-    "@storybook/testing-library": "^0.1.0",
-    "@storybook/theming": "^7.0.7",
-    "@sveltejs/vite-plugin-svelte": "^2.4.2",
-    "@types/prismjs": "^1.26.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.0",
-    "@typescript-eslint/parser": "^5.59.0",
-    "@viamrobotics/eslint-config": "^0.0.4",
-    "@viamrobotics/prettier-config": "^0.0.1",
-    "@viamrobotics/typescript-config": "^0.0.3",
-    "@vitejs/plugin-vue": "^4.1.0",
-    "classnames": "^2.3.2",
-    "concurrently": "^8.0.1",
-    "element-internals-polyfill": "^1.3.0",
-    "eslint": "^8.39.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-import-resolver-typescript": "^3.5.5",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prefer-arrow": "^1.2.3",
-    "eslint-plugin-sonarjs": "^0.19.0",
-    "eslint-plugin-storybook": "^0.6.11",
-    "eslint-plugin-svelte": "^2.31.0",
-    "eslint-plugin-tailwindcss": "^3.11.0",
-    "eslint-plugin-unicorn": "^46.0.0",
-    "mdi-svelte-ts": "^1.0.0",
-    "monaco-editor": "^0.37.1",
-    "postcss": "^8.4.23",
-    "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.1",
-    "prismjs": "^1.29.0",
-    "react": "^18.2.0",
-    "remark-gfm": "^3.0.1",
-    "storybook": "^7.1.0",
-    "svelte": "^4.0.5",
-    "svelte-check": "^3.4.3",
-    "svelte-preprocess": "^5.0.3",
-    "tailwindcss": "^3.3.1",
-    "typescript": "^5.1.6",
-    "vite": "4.3.1",
-    "vue": "^3.2.45",
-    "vue-tsc": "^1.2.0"
-  }
+  "packageManager": "pnpm@8.6.3"
```

Could bump this to latest while we're here

```suggestion
  "packageManager": "pnpm@8.6.10"
```

---

Updated, thanks for the suggestion